torchao/utils.py

```diff
@@ -22,20 +24,71 @@
 ]
 
 
+def _assert_and_get_unique_device(module: torch.nn.Module) -> Any:
```

Comment:
Let's also make sure to remove this function from the quantization_utils file cause now code is just duplicated

Reply:
Function is referenced from https://github.com/pytorch/pytorch/blob/9105d54c6b37099575c0059ef274c86c4dc80c57/torch/ao/quantization/utils.py#L711